wherehows-web/app/styles/components/dataset-acl-access/_dataset-acl-access.scss

```diff
@@ -0,0 +1,105 @@
+$success-color: rgb(19, 158, 38);
+$reject-color: rgb(224, 0, 34);
+$gray-color: rgb(234, 234, 234);
+
+.acl-permission {
+  &__header {
+    display: flex;
+    align-items: flex-start;
+    margin-top: item-spacing(3);
+  }
+
+  &__meta {
+    display: flex;
+    flex-direction: column;
+    margin-left: item-spacing(2);
+
+    &__header {
+      font-size: 18px;
+    }
+  }
+
+  &__meta strong {
+    font-size: 16px;
+  }
+
+  &__success {
+    color: $success-color;
+  }
+
+  &__reject {
+    color: $reject-color;
+  }
+}
+
+.acl-form {
+  list-style-type: none;
+  padding: item-spacing(5);
+  padding-top: item-spacing(3);
+  border: 1px solid $gray-color;
+  border-radius: item-spacing(1);
+
+  &__header {
+    display: block;
+    font-weight: 600;
+    font-size: 19px;
+  }
+
+  &__meta {
+    margin: 0;
+
+    &__container {
+      display: flex;
+      flex-direction: column;
+    }
+
+    &__header {
+      font-size: 18px;
+      margin-bottom: 0;
+      font-weight: 500;
+    }
+
+    &__content {
+      color: rgb(115, 115, 115);
+      font-weight: 400;
+      font-size: 16px;
+    }
+  }
+
+  &__content {
+    min-height: item-spacing(6) * 3;
+    outline: none;
+    width: item-spacing(9)*9;
+    overflow: auto;
+    padding: item-spacing(1);
+    background-color: rgb(245, 246, 248);
+    resize: none;
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.16), 0 0 0 1px rgba(0, 0, 0, 0.08);
+    transition: box-shadow 200ms cubic-bezier(0.4, 0, 0.2, 1);
+    border-radius: 2px;
+    border-color: #ededed;
+  }
+
+  &__actions {
+    margin: item-spacing(4) 0 0;
+
+    button {
+      margin: item-spacing(0 1 0 0);
+      outline: none;
+    }
+  }
+}
+
+.acl-table {
+  &__header {
+    font-weight: bold;
+  }
+
+  &__ownership {
+    color: #bfc0c1;
+  }
+}
+
+#acl-permissioninfo hr {
```

Comment:
As a rule we should never use id's as a style rule in css except with good cause.

> IDs shouldn't be used in selectors because these rules are too tightly coupled with the HTML and have no possibility of reuse. It's much preferred to use classes in selectors and then apply a class to an element in the page. Additionally, IDs impact your specificity and can lead to specificity wars.

https://github.com/CSSLint/csslint/wiki/Disallow-ids-in-selectors